CONTRIBUTING.md

```diff
@@ -65,3 +65,68 @@ Explain the problem and include additional details to help maintainers reproduce
 
 
 This text has been largely inspired by the Atom contributing notes, which amazingly complete
+
+
+## Styleguides
+
+These sections tend to bore people. But if you don't follow them, it slows
+down reviews from developers (and tends to tire/annoy them). Please give it
+a look over and feel free to ask questions.
+
+### Git Commit Messages
+
+Rather than duplicate a ton of great explanations, please read this site:
+http://chris.beams.io/posts/git-commit/
+
+Yes, it's long, but once you read and understand it once, it applies to
+every git project out there, not just GD.
+
+In case that site is broken, here is an archived link:
+http://web.archive.org/web/20160706012209/http://chris.beams.io/posts/git-commit/
```

Comment:
then the fallback is working ? :)

we've had links break before and have gaps before they're updated.  i think people understand at this point it's best effort.